source/cloud/cloud-administration/cloud-changelog.md

```diff
@@ -0,0 +1,86 @@
+# Mattermost Cloud Changelog
+
+This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
+
+## Release 2020-12-09
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the ability to mute categories with the experimental sidebar feature.
+ - Added support for multi-selection of channels for dragging and dropping between channels in the experimental sidebar feature.
+ - Group messages are now returned in the channel switcher when only first names are typed.
+ - /dnd command will only set a do not disturb status whatever the current user status is.
```

Comment:
This bullet point is unclear. Will the `/dnd` command change the user's current status to `do not disturb`? If so, what part of this functionality has changed as part of this release?

Reply:
Adjusted the wording.